docs/src/hohqmesh_tutorial.md

```diff
@@ -0,0 +1,481 @@
+# Unstructured meshes with HOHQMesh.jl
+
+Trixi.jl supports numerical approximations on unstructured quadrilateral meshes
+with the `UnstructuredMesh2D` mesh type.
+
+The purpose of this tutorial is to demonstrate how one can use the `UnstructuredMesh2D`
+functionality of Trixi.jl. This begins by running and visualizing an available unstructured
+quadrilateral mesh example. Then, the tutorial will demonstrate how one can
+conceptualize a problem with curved boundaries, generate
+a curvilinear mesh using the available software in the Trixi.jl ecosystem,
+and then run a simulation using Trixi.jl on said mesh.
+
+Unstructured quadrilateral meshes can be made
+with the [*High-Order Hex-Quad Mesh (HOHQMesh)*](https://github.com/trixi-framework/HOHQMesh)
+generator created and developed by David Kopriva.
+HOHQMesh is a mesh generator specifically designed for spectral element methods where elements can be larger (due to the high accuracy
```

Comment:
"larger" than what?

Reply:
compared to traditional finite element mesh generators. I can clarify this

Reply:
That would be nice :+1:  Not everyone reading this will have our background and add these words automatically